cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/NamespaceHttpHandlerTest.java

```diff
@@ -277,6 +279,14 @@ public void testDeleteAll() throws Exception {
     Assert.assertTrue(streamAdmin.exists(myStream));
     assertResponseCode(200, deleteNamespace(OTHER_NAME));
     Assert.assertFalse(streamAdmin.exists(myStream));
+
+    // Create the namespace again and deploy the application containing schedules.
```

Comment:
Yes. Thats correct. Unrecoverable reset deletes the app but not schedule. So deploying the app again fails with SchedulerException.